modules/ecs-service/cloud-map.tf

```diff
@@ -0,0 +1,71 @@
+module "cloudmap_namespace" {
+  for_each = { for service_connect in var.service_connect_configurations : service_connect.namespace => service_connect }
+
+  source  = "cloudposse/stack-config/yaml//modules/remote-state"
+  version = "1.5.0"
+
+  component = each.key
+
+  # we ignore errors because the namespace may be a name or an arn of a namespace for the service.
+  ignore_errors = true
+  context       = module.this.context
+}
+
+locals {
```

Comment:
can we merge locals here and on line 32?

Reply:
I'd prefer not to, they are for two different methods of connecting ecs services, either Service Discovery or Service Connect.